Breeze.UI/main.ts

```diff
@@ -12,8 +12,10 @@ const os = require('os');
 var apiProcess;
 
 let serve;
+let debug;
 const args = process.argv.slice(1);
 serve = args.some(val => val === "--serve");
+debug = args.some(val => val === "--debug");
```

Comment:
Hey @Daegalus,

The debug parameter is not necessary, because when you run the UI with npm start it will automatically add --serve.
It's sufficient to use if(serve) at these places. Can you please adjust this?

Rest looks good!